docs/sources/structure/_index.md

```diff
@@ -54,27 +54,132 @@ Depending on your product design and maturity, you may not need every topic:
+      </td>
+    </tr>
+    <tr>
+      <td><em>Manage alerts</em></td>
+      <td><a href=https://grafana.com/docs/grafana-cloud/alerting-and-irm/slo/manage/>Manage SLOs</a></td>
+      <td>
+        Specific to Grafana Ops products such as Alerting, OnCall, Incident, and SLO.
+        The word <em>alert</em> may be changed, depending upon the product.
+        If used with Grafana SLO, this then topic would be <em>Create SLO</em>.
+        Don't use with backend database products, such as Tempo and Loki.
+        Use <em>Alert</em> instead, and refer to an operational product for details.
+      </td>
+    <tr>
+    <tr>
+      <td><em>Integrate [with] &lt;PRODUCT&gt;</em> or <em>Send data</em></td>
+      <td><a href=https://grafana.com/docs/grafana-cloud/monitor-infrastructure/get-started-data/>Instrument and send data to Grafana Cloud</a>
+      <td>How to set up data integrations, product integrations, data sources, clients, plugins, and more.</td>
+    </tr>
+    <tr>
+      <td><em>Query data</em></td>
+      <td><a href=https://grafana.com/docs/tempo/latest/traceql/>Query with TraceQL</a>
+      <td>Query languages, query tools, and examples.</td>
+    </tr>
+    <tr>
+      <td><em>Visualize data</em></td>
+      <td><a href=https://grafana.com/docs/grafana/latest/dashboards/>Dashboards</a>
+      <td>Dashboard concepts and procedures. Link to the definitive source of dashboard documentation, rather than duplicating the information here.</td>
+    </tr>
+    <tr>
+      <td><em>Alert</em></td>
+      <td><a href=https://grafana.com/docs/loki/latest/alert/>Alerting and recording rules</a>
+      <td>
+        This topic level is used for pages that discuss alerting features, like alerting rules in Grafana Loki.
+        It provides a place for alerting content that's not specific to the Grafana Operations products.
+      </td>
+    </tr>
+    <tr>
+      <td><em>Manage &lt;PRODUCT&gt;</em></td>
+      <td><a href=https://grafana.com/docs/grafana-cloud/alerting-and-irm/oncall/manage/user-and-team-management/>Manage users and teams for Grafana OnCall</a>
+      <td>
+        Information about managing a Grafana Labs product.
+        For example, content in this topic helps you view, edit, and iterate on the Grafana product you installed.
+      </td>
+    </tr>
+    <tr>
+      <td><em>Monitor &lt;PRODUCT&gt;</em></td>
+      <td><a href=https://grafana.com/docs/mimir/latest/operators-guide/monitor-grafana-mimir/>Monitor Grafana Mimir</a>
+      <td>Information about using tools to monitor a Grafana Labs product.</td>
+    </tr>
+    <tr>
+      <td><em>References</em></td>
+      <td><a href=https://grafana.com/docs/mimir/latest/references/>Grafana Mimir references</a>
+      <td>
+        APIs, configuration references, SDKs, and more.
+        Material that's usually not procedural.
+      </td>
+    </tr>
+  </tbody>
+<table>
+
+#### Map the topic list to your table of contents
 
 ## Table of contents levels
```

Comment:
Should there be content here?

Reply:
Oh that's an accidental include, thanks for spotting!
```suggestion
## Table of contents levels
```